src/main/java/com/google/step/datamanager/DealManagerDatastore.java

```diff
@@ -105,4 +136,66 @@ public Deal updateDeal(Deal deal) {
     searchManager.putDeal(deal, new ArrayList<>());
     return readDeal(deal.id);
   }
+
+  /** Retrieves deals posted by _ followed by user */
+  private List<Deal> getDealsPublishedByFollowedRestaurantsOrUsers(
+      long userId, String fieldName, String filterAttribute) {
```

Comment:
I'm also not sure what's the difference between `fieldName` and `filterAttribute` used for